src/SprykerEco/Yves/Computop/Handler/PostPlace/ComputopPayuCeeSinglePaymentHandler.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Computop\Handler\PostPlace;
+
+use Generated\Shared\Transfer\ComputopPayuCeeSinglePaymentTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Shared\Kernel\Transfer\AbstractTransfer;
+
+class ComputopPayuCeeSinglePaymentHandler extends AbstractPostPlacePaymentHandler
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     * @param \Spryker\Shared\Kernel\Transfer\AbstractTransfer|\Generated\Shared\Transfer\ComputopPayuCeeSingleInitResponseTransfer $responseTransfer
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    protected function addPaymentToQuote(QuoteTransfer $quoteTransfer, AbstractTransfer $responseTransfer): QuoteTransfer
+    {
+        $payment = $quoteTransfer->getPayment();
+
+        if (!$payment) {
+            return $quoteTransfer;
+        }
+
+        if ($payment->getComputopPayuCeeSingle() === null) {
```

Comment:
* Please choose one way of handling nullable properties and use it.

Reply:
Done